slicingTool/runProjectSlicingTool.sh

```diff
@@ -26,14 +34,23 @@ if [[ -n $DB && -n $USER && -n $PASS ]]
 then
     if [[ $USER =~ "curator" ]]
     then
-	echo Dropping $DB ...
-	mysql -u$USER -p$PASS -e "drop database if exists $DB"
-#	mysql -u$USER -p$PASS -e "drop database if exists ${DB}_myisam"
-#	mysql -u$USER -p$PASS -e "create database ${DB}_myisam"
-#	mysql -u$USER -p$PASS -e "drop database if exists gk_central"
-#	mysql -u$USER -p$PASS -e "create database gk_central"
+        # Attempts to find database $DB and if successful (mysqlshow returns a zero exit status) it is backed up
+        # before being dropped
+        mysqlshow $DB > /dev/null 2>&1
+        if [[ $? == 0 ]]
+        then
+            echo Backing up $DB ...
+            mysqldump -u$USER -p$PASS $DB > $DB.dump
+        fi
+
+        echo Dropping $DB ...
+        mysql -u$USER -p$PASS -e "drop database if exists $DB"
+        #mysql -u$USER -p$PASS -e "drop database if exists ${DB}_myisam"
```

Comment:
Why are all these lines commented out? Can they be cleaned up?

Reply:
They were commented out before making changes to the script.  I fixed the mixed indentation of tabs and spaces.  I think they can be removed, though.

Reply:
Okay, I've taken them out since those databases aren't really impacted by the slicing tool.